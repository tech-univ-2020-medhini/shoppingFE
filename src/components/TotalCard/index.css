.total-card{
    display: flex;
    flex-direction: column;
    box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
    transition: 0.3s;
    height: 15rem;
    width: 15rem;
    border: groove;
    background-color: rgb(196,196,196);
    border-radius: 10%;
    padding: 1rem;
}
.total-card-text{
    display: flex;
    flex-direction: row;
    justify-content: space-evenly;
}
.rs{
    font-size: xx-small;
}
.total{
    font-size: medium;
}
.price{
    font-size: xx-large;
}
.total-card:hover{
    box-shadow: 0 8px 16px 0 rgba(0,0,0,0.2);
}
