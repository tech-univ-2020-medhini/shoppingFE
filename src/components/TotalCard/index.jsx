import React from 'react';
import PropTypes from 'prop-types';
import './index.css';
import Button from './components/Button';

function TotalCard(props) {
  const { total } = props;

  return (
    <div className="total-card">
      <div className="total">
        Total
      </div>
      <div className="rs">
        RS
      </div>
      <div className="price">
        {total}
      </div>
      <hr />
      <Button />

    </div>

  );
}
TotalCard.propTypes = {
  total: PropTypes.number.isRequired,
};

export default TotalCard;
