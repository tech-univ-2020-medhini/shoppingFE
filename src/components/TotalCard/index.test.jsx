import React from 'react';
import { render, fireEvent } from '@testing-library/react';
import TotalCard from './index';


describe('the cart table', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<TotalCard />);

    expect(asFragment()).toMatchSnapshot();
  });
});
