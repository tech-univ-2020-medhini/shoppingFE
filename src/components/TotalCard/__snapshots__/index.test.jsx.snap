// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the total card should render correctly 1`] = `
<DocumentFragment>
  <div
    class="total-card"
  >
    <div
      class="total-card-text"
    >
      <div
        class="total"
      >
        Total
      </div>
      <div
        class="rs"
      >
        RS
      </div>
      <div
        class="price"
      />
    </div>
    <hr />
    <div
      class="total-checkout"
    >
      <button
        class="button-yellow"
        data-testid="test-btn"
        type="button"
      >
        Checkout
      </button>
    </div>
  </div>
</DocumentFragment>
`;
